src/test/java/org/candlepin/subscriptions/marketplace/MarketplaceProducerTest.java

```diff
@@ -53,9 +53,8 @@ void testMarketplaceProducerRetry() throws Exception {
         when(marketplaceService.submitUsageEvents(any())).thenThrow(SubscriptionsException.class);
 
         var usageRequest = new UsageRequest();
-        assertThrows(SubscriptionsException.class, () ->
-            marketplaceProducer.submitUsageRequest(usageRequest)
-        );
+
+        marketplaceProducer.submitUsageRequest(usageRequest);
 
```

Comment:
I think you should still test the case where an exception is thrown.

Reply:
@dmckoske with this change, it's no longer possible for `submitUsageRequest` to throw any exception; see https://github.com/RedHatInsights/rhsm-subscriptions/blob/906b7b03c5157d8e88c7ce93250f8d07f7581cf6/src/main/java/org/candlepin/subscriptions/marketplace/MarketplaceProducer.java#L93-L98

Reply:
Oh, hello.  Okay, good.